lib/optimize.js

```diff
@@ -155,7 +170,7 @@ var optimizer = {
 
             if (reducer === 'count_unique') {
                 aggrs[target] = {
-                    raw: 'COUNT(distinct "' + arg + '") as ' + target
+                    raw: 'COUNT(distinct ' + arg + ') as ' + target
```

Comment:
I found that mysql was failing the test when using this raw sql. When I took out the quotes it worked for all dialects.